### ISSUE TITLE
Call to undefined method App\Installer\Languages::getToInstall()

### ISSUE THREAD

**Issue by hilligsmann** [YetiForceCompany/YetiForceCRM#12036]
Hello, 

I just uploaded the YetiForceCRM-5.2.0-complete. When trying to acceed the global settings inside the app, I get the error Call to undefined method App\Installer\Languages::getToInstall().

I then updated to 5.1.0_to_5.2.0_v5 (overwrite in FTP) and added the languages de_DE and fr-FR (used by myself as a user) to the directory. 

Bug remains.

Could you please help?

**Comment by hilligsmann** [comment]
Hello, problem remains. Could you help?

**Comment by vovpff** [comment]
At first: your installation was unsuccessful. Next question: what mean overwrite in FTP update from 5.1 to 5.2 if you was installed 5.2?

**Comment by hilligsmann** [comment]
Hello, thanks for reply. I made sure I was using the last version, uploading the 5.1.0_to_5.2.0_v5 file to FTP. 

What should I do?

**Comment by vovpff** [comment]
Update should be installed trough admin panel. Not just uploaded somethere. Check your server configuration - I'm sure what you have wrong configuration...

**Comment by mariuszkrzaczkowski** [comment]
@hilligsmann 
![github](https://user-images.githubusercontent.com/10198654/69216472-99077e00-0b6c-11ea-91ed-94ee73f9360e.jpg)

**Comment by dldnet** [comment]
Exactly the same problem on a fresh install 5.2 and  service pack 19 applied correctly. Impossible to go to system settings (upper right button)

**Comment by dldnet** [comment]
Did you find any solution ?

**Comment by mariuszkrzaczkowski** [comment]
read the rules for reporting bugs.
you did not attach the required information, there is no complete description to understand your problem.
Nobody will help you if you do not send full information

**Comment by dldnet** [comment]
All these info are not needed, sorry.
So, here is the solution:
The file in your release 5.2 app/Installer/Languages.php is outdated. 
It doesn't include the function getToInstall()

So, just copy the same file app/Installer/Languages.php from the GitHub repository and replace the old one.

And everything is ok.
So, I suggest to add it in the next service pack.

Just say thank you ;-)

**Comment by mariuszkrzaczkowski** [comment]
version 5.2 has been installed many times and no one has seen or reported this problem.  that's why a detailed description is needed

**Comment by dldnet** [comment]
As said: I have downloaded a fresh install 5.2 from GitHub Release (zip). It contains the Languages.php file outdated. Then, I have applied the latest Service Pack but it doesn't contain any change on the Languages.php file.
Just copy the latest version of Languages.php from GitHub solve the problem.
I have nothing to send, the only thing to do is to compare the file Languages.php in the 5.2 release zip and the latest version. 
That's not a local problem, that's a packaging problem on GitHub.
